lib/ui/popover/popover.js

```diff
@@ -44,12 +44,26 @@
     this.destroy = function() {
       $element.popover('destroy');
     };
+
+    if($scope.showOnLoad) {
+      this.show();
+      if($scope.delay && $scope.delay.hide) {
+        $timeout(this.hide, $scope.delay.hide);
+        return;
+      }
+      // If no delay is found or cannot be parsed, set a default timeout so that the popover doesn't stick around forever
+      $timeout(this.hide, 10000);
```

Comment:
We definitely don't depend on any data changes for the hide function, I'll add the false flag. I also agree with your second comment, I'll add a config provider.